### PR TITLE
vdk-control-service: fix helm template

### PR DESCRIPTION
**Pull request by murphp15**
### Why
In a previous commit a helm chart but was submitted. 
### What
This makes a small correction as repo type is no longer used 

### How was it tested
I have added in a test step in the PR pipeline to ensure this never happens again. 

Signed-off-by: murphp15 <murphp15@tcd.ie>

### CODE REVIEW

File: projects/control-service/cicd/test-testing-pipelines-service.sh
Comment (tozka) [inline projects/control-service/cicd/test-testing-pipelines-service.sh:40]: I like the test but not the code duplication now when we change the helm arguments we have to do it in two places

**Review by tozka — CHANGES_REQUESTED** [review]

**Comment by gageorgiev** [comment]
I'm not sure I understand what the issue previously was.

File: projects/control-service/projects/helm_charts/pipelines-control-service/templates/_helpers.tpl
Comment (murphp15) [inline projects/control-service/projects/helm_charts/pipelines-control-service/templates/_helpers.tpl:214]: @gageorgiev  there was no property called .Values.image.registryType. That is why it was failing.

File: projects/control-service/cicd/test-testing-pipelines-service.sh
Comment (murphp15) [reply projects/control-service/cicd/test-testing-pipelines-service.sh]: the template function is checking the validity of the possible helm charts. 
I have removed all properties for clarity.
This should catch the bug I previously introduced.

**Comment by mivanov1988** [comment]
can't it be used `helm lint` for the verification?

The output of `helm lint`:
```
==> Linting .
[INFO] Chart.yaml: icon is recommended
[ERROR] templates/: template: pipelines-control-service/templates/secret_pull_pipelines_control_service_img.yaml:1:8: executing "pipelines-control-service/templates/secret_pull_pipelines_control_service_img.yaml" at <include "shouldCreatePipelinesControlServiceDockerRepoSecret" .>: error calling include: template: pipelines-control-service/templates/_helpers.tpl:214:14: executing "shouldCreatePipelinesControlServiceDockerRepoSecret" at <eq .Values.image.registryType "generic">: error calling eq: incompatible types for comparison
[WARNING] /home/mivanov/Development/Projects/github/versatile-data-kit/projects/control-service/projects/helm_charts/pipelines-control-service: chart directory is missing these dependencies: common,cockroachdb

Error: 1 chart(s) linted, 1 chart(s) failed
```

**Review by tozka — APPROVED** [review]
I think we should prioritize fixing the pipeline on main https://gitlab.com/vmware-analytics/versatile-data-kit/-/commits/main as soon as possible. Usually the easiest way to achieve that is to revert temporarily. 

Otherwise, any other development work is blocked on the control service including any other emergency releases, hotfixes, etc. 

So let's merge this if it fixes it. And you can address any comments in follow-up review

**Review by mivanov1988 — APPROVED** [review]

**Review by mivanov1988 — APPROVED** [review]

**Pull request merged**
